cedar-policy-validator/src/cedar_schema/to_json_schema.rs

```diff
@@ -165,25 +195,63 @@ impl TryFrom<Namespace> for json_schema::NamespaceDefinition<RawName> {
         let (entity_types, action, common_types) = into_partition_decls(n.decls);
 
         // Convert entity type decls, collecting all errors
-        let entity_types = collect_all_errors(entity_types.into_iter().map(convert_entity_decl))?
-            .flatten()
-            .collect();
+        let entity_types = collect_all_errors(entity_types.into_iter().map(|et| {
+            let et = annotations::Annotated::try_from(et)
+                .map_err(|err| ToJsonSchemaErrors::from(ToJsonSchemaError::from(err)))?;
+            convert_entity_decl(et.data).map(|i| {
```

Comment:
IMO it might be nicer if `convert_entity_decl` took the annotated entity type as it argument and returned the converted decl with annotations